docs/3 Scaffold a Blockchain/5 Message Scaffold.md

```diff
@@ -0,0 +1,28 @@
+# Message Scaffold
+
+Cosmos SDK messages are used to modify the state of a blockchain. Messages are bundled into transactions, broadcasted transactions are bundled into blocks and blocks make a blockchain.
+
+Starport supports scaffolding Cosmos SDK messages.
+
+```
+starport message [name] [field1] [field2] ... [flags]
+```
+
+Files and directories created and modified by scaffolding:
+
+* `proto`: message type
+* `x/module_name/keeper`: gRPC message server
```

Comment:
or we build this directory and what goes here? not sure how much detail we need

Reply:
We add a couple of files into the `keeper` directory. I don't want to be very specific about which files we add, because this changes, and I don't want docs to go out-of-date quickly.